src/monitor-view-events.js

```diff
@@ -0,0 +1,53 @@
+// DOM Refresh
```

Comment:
hmm, maybe this could be moved to `src/utils/monitor-view-events`... it's not a primary class